### PR TITLE
[DOC-194] [DOC-195] [2.20.2] VM/K8S task retryability 

### PR DESCRIPTION
**Pull request by aishwarya24**

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+![Retry task](/images/yp/retry-task.png)
+
+A retry of the original task reruns the individual steps of the task in an idempotent fashion. This should resolve a failure if it was caused by a temporary issue.
+
+For example, a quota limit may prevent a universe scale up a task causing it to fail. When the quota limit is adjusted to the right value and the task is retried, it should succeed. If the underlying issue is more permanent, contact {{% support-platform %}}.
+
+Critical task failures such as universe scale up or down, placement changes, software upgrades, and flags changes disable other critical tasks from being run on the universe. It is recommended to retry the failed task to completion before attempting other operations on the universe, or contact {{% support-platform %}} if the task continues to fail.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md:22]:
```suggestion
Critical task failures, such as scaling a universe, placement changes, software upgrades, and flag changes block other critical tasks from being run on the universe. It is recommended to retry the failed task to completion before attempting other operations on the universe, or contact {{% support-platform %}} if the task continues to fail.
```

File: docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+linkTitle: Retry a universe task
+description: Use YugabyteDB Anywhere to retry failed tasks.
+menu:
+  stable_yugabyte-platform:
+    identifier: retry-task
+    parent: manage-deployments
+    weight: 75
+type: docs
+---
+
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md:13]: What about Tasks tab? Tasks are listed there; can you retry from there as well?

Show both methods, and introduce the Tasks tab generally as a centralized spot for checking this stuff

File: docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+In case of most task failures in YugabyteDB Anywhere (YBA) universes (VMs or Kubernetes), you can retry the tasks from the YBA UI or via an [equivalent API](https://api-docs.yugabyte.com/docs/yugabyte-platform/68aaf7829e04f-retry-a-universe-task).
+
+You can retry a task using one of the following ways:
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md:16]:
```suggestion
To retry a failed task, do one of the following:
```

File: docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+    ![Retry task](/images/yp/retry-task1.png)
+
+- Navigate to **Tasks** in the YBA UI, click **See Details** corresponding to the failed universe task, and click **Retry Task**.
+
+    ![Retry task](/images/yp/retry-task2.png)
+
+A retry of the original task reruns the individual steps of the task in an idempotent fashion. This should resolve a failure if it was caused by a temporary issue.
+
+For example, a quota limit may prevent a universe scale up a task causing it to fail. When the quota limit is adjusted to the right value and the task is retried, it should succeed. If the underlying issue is more permanent, contact {{% support-platform %}}.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md:28]:
```suggestion
For example, a quota limit may prevent a universe scale up task, causing it to fail. When the quota limit is adjusted to the right value and the task is retried, it should succeed. If the underlying issue is more permanent, contact {{% support-platform %}}.
```

File: docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Critical task failures, such as scaling a universe, placement changes, software upgrades, and flag changes block other critical tasks from being run on the universe. It is recommended to retry the failed task to completion before attempting other operations on the universe, or contact {{% support-platform %}} if the task continues to fail.
+Note that some operations like taking [universe backups](../../back-up-restore-universes/), or creating [support bundles](../../troubleshoot/universe-issues/#use-support-bundles) can always be attempted even after such failures.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md:31]:
```suggestion

Note that some operations, such as taking [universe backups](../../back-up-restore-universes/) or creating [support bundles](../../troubleshoot/universe-issues/#use-support-bundles), can always be attempted even after such failures.
```

**Review by ddhodge — APPROVED** [review]
copy to preview

**Comment by aishwarya24** [comment]
> copy to preview

This is available only from 2.20.2. So I have it only in stable.

**Pull request merged**